pkg/apis/sources/v1alpha1/kafka_types.go

```diff
@@ -95,6 +95,17 @@ type KafkaSourceSpec struct {
 	// +optional
 	// Needed for supporting round-tripping
 	CloudEventOverrides *duckv1.CloudEventOverrides `json:"ceOverrides,omitempty"`
+
+	// Number of desired consumers. Defaults to 1.
+	//
+	// A consumer is an unit for measuring resource allocation,
+	// for example the number of pods, or the allocated bandwidth.
```

Comment:
The allocated bandwidth?

What about "the number of consumers running in the consumer group"?

Reply:
vaguer is a better :-) More consumers = faster, that's the only thing we need to know at the level.

Reply:
+1 for the wording you just committed